apps/wallet/src/ui/app/components/accounts/EditNickname.tsx

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useZodForm } from '@mysten/core';
+import { useCallback } from 'react';
+import toast from 'react-hot-toast';
+import { useNavigate, useParams } from 'react-router-dom';
+import { z } from 'zod';
+import { useAccounts } from '../../hooks/useAccounts';
+import { useBackgroundClient } from '../../hooks/useBackgroundClient';
+import { Button } from '../../shared/ButtonUI';
+import { Form } from '../../shared/forms/Form';
+import { TextField } from '../../shared/forms/TextField';
+import {
+	Dialog,
+	DialogContent,
+	DialogHeader,
+	DialogTitle,
+	DialogDescription,
+} from '_src/ui/app/shared/Dialog';
+
+const formSchema = z.object({
+	nickname: z.string().nonempty('Required'),
```

Comment:
It there a trim transform we can do here? To avoid setting spaces as a nickname (fine as is no need to spend much time just wondering if there is something we can easily do)